src/Splitio-net-core/Services/Shared/Classes/ConfigService.cs

```diff
@@ -89,6 +93,18 @@ private int GetMinimunAllowed(int value, int minAllowed, string configName)
 
             return value;
         }
+
+        private int GetImpressionRefreshRate(ImpressionModes impressionModes, int? impressionsRefreshRate)
+        {
+            switch (impressionModes)
```

Comment:
What will happen is the user sends a string that doesn't follow the expected string? I meant, camelcase, lowercase? Are you converting the passed config into lower or camel case to check?